UPGRADING.md

```diff
@@ -1,6 +1,131 @@
+
+Starting Bugsnag via a GameObject has been removed in favor of Bugsnag starting automatically just before the first scene is loaded. 
+
+To setup your Bugsnag configuration, please open the window by going to Window/Bugsnag/Settings. This will show the Bugsnag settings menu where you can add your API key and other settings.
+
+It is still possible to initialise in code with `Bugsnag.Start(Configuration config)`.
+
+### Metadata
+
+#### Additions
+
+The following methods have been added to the `Bugsnag` class: 
+
+| Property/Method                                                    | Usage                                                             |
+| ------------------------------------------------------------------ | ----------------------------------------------------------------- |
+| `AddMetadata(string section, string key, object value)`            | Add a specific value by key to a section of metadata LINK_DOCS    | 
+| `AddMetadata(string section, Dictionary<string, object> metadata)` | Add a section by name to the metadata metadata LINK_DOCS          | 
+| `ClearMetadata(string section)`                                    | Clear a section of metadata LINK_DOCS          | 
+| `ClearMetadataClearMetadata(string section, string key)`           | Clear a specific entry in a metadata section by key LINK_DOCS          | 
+| `Dictionary<string, object> Bugsnag.GetMetadata(string section)`           | Get an entire section from the metadata as a Dictionary LINK_DOCS          | 
+| `object Bugsnag.GetMetadata(string section, string key)`           | Get an specific entry from the metadata as an object LINK_DOCS          | 
+
+#### Deprecations
+
+The following property has been removed from the `Bugsnag` client:
+
+| v5.x API                                                           | v6.x API                                                          |
+| ------------------------------------------------------------------ | ----------------------------------------------------------------- |
+| `Bugsnag.Metadata`                                                 | Deprecated - no longer public API.                                |
+
+
+### Breadcrumbs
+
+#### Deprecations
+
+The following properties/methods have been removed from the `Bugsnag` client:
+
+| v5.x API                                                           | v6.x API                                                          |
+| ------------------------------------------------------------------ | ----------------------------------------------------------------- |
+| `Bugsnag.LeaveBreadcrumb(message)`                                 | Replaced by `Bugsnag.LeaveBreadcrumb(message, metadata, type)`    |
+| `Bugsnag.LeaveBreadcrumb(message, type, metadata)`                 | Replaced by `Bugsnag.LeaveBreadcrumb(message, metadata, type)`    |
+| `Bugsnag.LeaveBreadcrumb(breadcrumb)`                              | Replaced by `Bugsnag.LeaveBreadcrumb(message, metadata, type)`    |
```

Comment:
The second and third arguments are optional, so I don't think it's true to say that `LeaveBreadcrumb(message)` is deprecated?

Reply:
Fixed 👍